tests/acceptance/features/apiSpaces/lockSpaces.feature

```diff
@@ -0,0 +1,93 @@
+@api @skipOnOcV10
+Feature: lock
+  # Note: This Feature includes all the tests from core (apiWebdavLock suite) related to /Shares since in core no implementation is there for space Shares Jail
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+      | Carol    |
+    And user "Alice" has created folder "PARENT"
+    And user "Brian" has created folder "PARENT"
+    And user "Carol" has created folder "PARENT"
+    And using spaces DAV path
+
+
+  Scenario Outline: lock should propagate correctly when uploaded to a reshare that was locked by the owner
+    Given user "Alice" has shared folder "PARENT" with user "Brian"
+    And user "Brian" has accepted share "/PARENT" offered by user "Alice"
+    And user "Brian" has shared the following entity "PARENT" inside of space "Shares Jail" with user "Carol" with role "editor"
+    And user "Carol" has accepted share "/PARENT" offered by user "Brian"
+    And user "Alice" has locked folder "/PARENT" inside space "Personal" setting the following properties
+      | lockscope | <lock-scope> |
+    When user "Carol" uploads a file "filesForUpload/textfile.txt" inside space "Shares Jail" to "PARENT/textfile.txt" using the WebDAV API
```

Comment:
```suggestion
    When user "Carol" uploads a file inside space "Shares Jail" with content "uploaded content" to "PARENT/textfile.txt" using the WebDAV API
```
We have a similar step implemented in `SpacesContext.php`. Can't we use the existing one or does your have a different purpose than the existing one?

Reply:
it uploads files from local system but may be we can use the existing step too. Thanks